frontend/client-search/client-list/client-search-input.component.tsx

```diff
@@ -34,17 +40,93 @@ export default function ClientSearchInput(props: ClientSearchInputProps) {
           autoFocus={props.autoFocus}
           ref={inputRef}
         />
-        <button type="submit" className="primary" disabled={props.disabled}>
-          Search
-        </button>
+        {!showingAdvancedSearch && (
+          <>
+            <button
+              type="button"
+              className="secondary"
+              disabled={props.disabled}
+              onClick={() => setShowingAdvancedSearch(true)}
+            >
+              Advanced
+            </button>
+            <button type="submit" className="primary" disabled={props.disabled}>
+              Search
+            </button>
+          </>
+        )}
       </form>
+      {showingAdvancedSearch &&
+        props.advancedSearchRef &&
+        ReactDOM.createPortal(
+          <form
+            className="advanced-search"
+            {...scope}
+            autoComplete="password"
```

Comment:
Is this for preventing autocomplete? If so, should this be
autoComplete="new-password"
?

Reply:
Good catch